wpeverest/everest-forms
# Fix - Attribute issue in Multipart

### All Submissions:



### Changes proposed in this Pull Request:
This PR fixed the multipart submit button and re captcha visible in every page.

### How to test the changes in this Pull Request:

1. Create From.
2. Enable Multipart  and re captcha in form.
3. verify submit button and re captcha showing at last part.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Attribute issue in Multipart
